packages/quick_actions/lib/quick_actions.dart

```diff
@@ -36,7 +33,16 @@ class ShortcutItem {
 
 /// Quick actions plugin.
 class QuickActions {
-  const QuickActions();
+  factory QuickActions() => _instance;
+
+  @visibleForTesting
+  QuickActions.private(MethodChannel channel) : _kChannel = channel;
+
+  static final QuickActions _instance = QuickActions.private(
+    const MethodChannel('plugins.flutter.io/quick_actions'),
+  );
+
+  final MethodChannel _kChannel;
```

Comment:
Let's keep the _KChannel as the constant for init by default and give this instance member a different name. We usually use a name started with k only for contants.
How about:
```suggestion
  final MethodChannel channel;
```